### PR TITLE
Use XML NPTG data

### PR DESCRIPTION
**Pull request by ljones92**

### CODE REVIEW

**Review by alainajaffer — DISMISSED** [review]

File: packages/ref-data-uploaders/nptg-uploader/index.ts [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    const nptgJson = (await parseStringPromise(nptgString, {
+        explicitArray: false,
+    })) as Record<string, unknown>;
+
+    const { adminAreas, localities } = nptgSchema.parse(nptgJson);
+
+    await setupTables(dbClient);
+
+    await uploadAdminAreas(adminAreas, dbClient);
+    await uploadLocalities(localities, dbClient);
```
Comment (alvak1) [inline packages/ref-data-uploaders/nptg-uploader/index.ts]: Could promise.all be used to upload admin areas and localities in parallel

**Comment by alvak1** [comment]
Add to makefile configs for test-nptg-uploader  and upload-nptg-file

File: packages/ref-data-uploaders/nptg-uploader/index.ts (hunk 1 above)
Comment (ljones92) [reply packages/ref-data-uploaders/nptg-uploader/index.ts]: Done

**Review by alvak1 — APPROVED** [review]
Approve

**Pull request merged**